src/Compilers/CSharp/Portable/BoundTree/UnboundLambda.cs

```diff
@@ -328,12 +328,13 @@ internal partial class UnboundLambda
             ImmutableArray<TypeWithAnnotations> types,
             ImmutableArray<string> names,
             bool isAsync,
-            bool hasErrors = false)
```

Comment:
note: there appear to be no callers that provide this value.  so it's always false.  @jcouv would you like me to remove it entirely?

Reply:
Have removed it.  Thanks!